Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,45 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// A Boolean that is `true` if peer SSL certificates are to be verified.
+    ///
+    /// You can bypass verification by setting this property to `false` (not recommended).
+    public static var verifyPeerSSLCertificate = true
+    /// The URL representing a PEM-encoded certificate authority file to be used for
+    /// peer SSL certificate verification. Setting a certificate authority file takes
+    /// precedence over the value of verifyPeerSSLCertificate.
+    ///
+    /// See:
+    /// - https://curl.haxx.se/docs/sslcerts.html
+    /// - https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL? {
```

Comment:
It’s Linux specific really where libcurl is being used but only essential on Android where the CA root certificates don't work with OpenSSL 1.0. Otherwise, no https: requests work. http://www.javacms.tech/questions/1728116/how-to-make-ssl-peer-verify-work-on-android